woocommerce/woocommerce
# Update task list documentation and example

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/35234.

This PR updates the task list documentation and example of "add task".



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Review the changes [onboarding-tasks.md](https://github.com/woocommerce/woocommerce/blob/dev/update-task-list-docs-example/plugins/woocommerce-admin/docs/features/onboarding-tasks.md)
2. Run `WC_EXT=add-task pnpm --filter=woocommerce/client/admin example`
3. Include ./woocommerce/plugins/add-task in your `.wp-env.json` or `.wp-env.override.json` and restart the WordPress instance
4. Go to `WooCommerce > Home`
5. Observe that "My Task" is shown in set up task list.
